### ISSUE TITLE
7 days to die console connexion script [BUG]

### ISSUE THREAD

**Issue by tristelune** [GameServerManagers/LinuxGSM#3888]
When activating 7dtd console with a password in the (serverconfig.xml) : edit for correction in the serverfiles/sdtdserver.xml

we set the line to true : 
property name="TelnetEnabled"                                  value="true"

and the password in : 
property name="TelnetPassword"                                 value="password"

When running the command to stop the server the script is failing to connect to the console to stop the server and kill it.

After debuging i found out that the script do not send the password set in the config file but the value of TelnetEnabled (so it send true instead of the password during the logon process)

to verify this behavior i configured the password with true:
property name="TelnetPassword"                                 value="true"  

When running the command to stop the server the script is now able to connect to the console to stop the server properly.

So some where in the code the TelnetEnabled var is used instead of the TelnetPassword

So basically we can't set a secure password to have the stop script be able to connect to the console, we have to put true all the time.

**Comment by tristelune** [comment]
To complement the var : "ControlPanelEnabled"  is set to false in my serverconfig.xml file

**Comment by tristelune** [comment]
> Potential duplicates:
> 
>     * [[[BUG] Cant bind IP to 7 days to die #3860](https://github.com/GameServerManagers/LinuxGSM/issues/3860)] [BUG] Cant bind IP to 7 days to die (70.24%)

Not duplicate, not the same issue

**Comment by h3o66** [comment]
afaik per default the config `serverfiles/sdtdserver.xml` is used and not the `serverconfig.xml` in serverfiles.
Please double check this on your side.

**Comment by tristelune** [comment]
sorry i replied by mail thinking it will add it to the tread, but no it didn't

Yes i'm talking about serverconfig.xml but of course i'm using serverfiles/sdtdserver.xml to configure the game overwise no other settings will apply (server password to connect, map selection, difficulty setting, drop on death,  motd, etc...) so yes the rcon password is indeed configured in the serverfiles/sdtdserver.xml file

sorry for the confusion

**Comment by tristelune** [comment]
I confirm that the settings are Indeed set in serverfiles/sdtdserver.xml config files otherwise server password, difficulty settings, server name, motd,maps, configuration would not apply on my server, I m hosting a 7dtd server since 4 years with lgsm.
________________________________

**Comment by dgibbs64** [comment]
7d2d recent update is not release and this should not be resolved
